src/uk/ac/open/audio/streaming/StreamPlayerUI.java

```diff
@@ -221,26 +219,47 @@ public void keyPressed(KeyEvent e) {
 				if (e.getKeyCode() == KeyEvent.VK_ENTER)
 				{
 					pause();
+					StreamPlayerUI.this.handleplaypauseenteredevent();
 		        }
 			}
 		});
 
 		inner.addFocusListener(new FocusListener() {
 			public void focusGained(FocusEvent e) {
-					progress.setPauseImage(StreamPlayerUI.this.pauseImage);
+	            StreamPlayerUI.this.handleplaypauseenteredevent();
 	        }
 
             public void focusLost(FocusEvent e) {
-              if (progress.getpauseFlag() == true)
-              {
-            	  progress.setPauseImage(StreamPlayerUI.this.pauseImage);
-			  } else {
-				   progress.setPauseImage(null);
-			   }
+	            StreamPlayerUI.this.handleplaypauseexitedevent();
             }
 		});
 	}
 
+	/**
+	 * function to handle play/pause image
+	 * on focus entered event
+	 * **/
+	private void handleplaypauseenteredevent() {
```

Comment:
Function name should use camelCalse, handlePlayPause etc. Also the javadoc here ends '* **/' instead of just '**/'.